src/index.html

```diff
@@ -46,7 +46,9 @@
       return "{{ REGISTRY }}".toLowerCase() === "true" || true ? true : false;
     };
     var stylingEnabled = function(){
-      return "{{ LOOM_STYLING_ENABLED }}".toLowerCase() === "true" ? true : false;
+      var perm = "{{ perms.layers.change_layer_style }}".toLowerCase() === "true" ? true : false;
```

Comment:
Would this apply if the user starts with an empty map and adds layers via the search modal?

Reply:
This doesn't check on a layer-by-layer basis, it only checks if they are a user who has the permissions to edit layer styles.